### ISSUE TITLE
Game crashes after completing the prep challange 

### ISSUE THREAD

**Issue by Kalik56** [CookiePLMonster/SilentPatchBully#95]
Every time I play the prep challenge it crashes at the end, it happens when FPS is 60 o 30. I even tried at different i game hours and it still crashes

[Bully.exe.20220525225910.zip](https://github.com/CookiePLMonster/SilentPatchBully/files/8775948/Bully.exe.20220525225910.zip)

**Comment by Xsamurai1337** [comment]
Yeah I had this same problem, the only exception is that my game never worked again.

I mean, congratulations to the people who are trying to fix this horrible game, the effort is trully appreciated, but at this point in time we all know the truth. There is no group on this planet who would be able to do such a job, Bully SE is 100% beyond repairing.
